CITATION.rst

```diff
@@ -12,7 +12,7 @@ If you publish results using Firedrake, we would be grateful if you would cite t
 
   @manual{FiredrakeUserManual,
     title        = {Firedrake User Manual},
-    author       = {David A. Ham and Paul H. J. Kelly and Lawrence Mitchell and Colin J. Cotter and Robert C. Kirby and Koki Sagiyama and Nacime Bouziani and Thomas J. Gregory and Jack Betteridge and Daniel R. Shapero and Reuben W. Nixon-Hill and Connor J. Ward and Patrick E. Farrell and Pablo D. Brubeck and India Marsden and Daiane I. Dolci and Joshua Hope-Collins and Sophia Vorderwuelbecke and Thomas H. Gibson and Miklós Homolya and Tianjiao Sun and Andrew T. T. McRae and Fabio Luporini and Alastair Gregory and Michael Lange and Simon W. Funke and Florian Rathgeber and Gheorghe-Teodor Bercea and Graham R. Markall},
+    author       = {David A. Ham and Paul H. J. Kelly and Colin J. Cotter and Robert C. Kirby and Koki Sagiyama and Jack Betteridge and Daniel R. Shapero and Connor J. Ward and Patrick E. Farrell and Pablo D. Brubeck and India Marsden and Daiane I. Dolci and Joshua Hope-Collins and Umberto Zerbinati and Lawrence Mitchell and Reuben W. Nixon-Hill and Nacime Bouziani and Sophia Vorderwuelbecke and Thomas H. Gibson and Miklós Homolya and Tianjiao Sun and Andrew T. T. McRae and Fabio Luporini and Alastair Gregory and Michael Lange and Simon W. Funke and Florian Rathgeber and Gheorghe-Teodor Bercea and Graham R. Markall},
```

Comment:
We need to not update the manual authors automatically because the manual is a published document. We should, however, publish a new edition now with current authors.